src/python/grpcio/grpc/_cython/_cygrpc/aio/iomgr/iomgr.pyx.pxi

```diff
@@ -121,7 +122,7 @@ cdef grpc_error* asyncio_socket_listen(grpc_custom_socket* grpc_socket) with gil
     return grpc_error_none()
 
 
-def _asyncio_apply_socket_options(object s, so_reuse_port=False):
+def _asyncio_apply_socket_options(object s):
```

Comment:
`s/s/socket_options/`

Reply:
Changed to `socket`, since this is a `Socket` object.